### ISSUE TITLE
CUDA_SETUP: WARNING! libcudart.so not found in any environmental path.

### ISSUE THREAD

**Issue by duskfallcrew** [TimDettmers/bitsandbytes#169]
===================================BUG REPORT===================================
Welcome to bitsandbytes. For bug reports, please submit your error trace to: https://github.com/TimDettmers/bitsandbytes/issues
For effortless bug reporting copy-paste your error into this form: https://docs.google.com/forms/d/e/1FAIpQLScPB8emS3Thkp66nvqwmjTEgxp8Y9ufuWTzFyr9kJ5AoI47dQ/viewform?usp=sf_link
================================================================================
CUDA_SETUP: WARNING! libcudart.so not found in any environmental path. Searching /usr/local/cuda/lib64...
CUDA SETUP: CUDA runtime path found: /usr/local/cuda/lib64/libcudart.so
CUDA SETUP: Highest compute capability among GPUs detected: 7.5
CUDA SETUP: Detected CUDA version 118
CUDA SETUP: Loading binary /usr/local/lib/python3.8/dist-packages/bitsandbytes/libbitsandbytes_cuda118.so...
use 8-bit AdamW optimizer | {}
override steps. steps for 20 epochs is / 指定エポックまでのステップ数: 2000
running training / 学習開始
  num train images * repeats / 学習画像の数×繰り返し回数: 400
  num reg images / 正則化画像の数: 0
  num batches per epoch / 1epochのバッチ数: 100
  num epochs / epoch数: 20
  batch size per device / バッチサイズ: 4
  total train batch size (with parallel & distributed & accumulation) / 総バッチサイズ（並列学習、勾配合計含む）: 4
  gradient accumulation steps / 勾配を合計するステップ数 = 1
  total optimization steps / 学習ステップ数: 2000

Weirdly after this it let me still train it?

**Comment by FinaBro69** [comment]
go to (Training parameters) you can see is (AdamW8bit change to AdamW)
and (Gradient checkpointing, Memory efficient attention) is two also tick it

Hope it helps you

**Comment by saikaboty** [comment]
> go to (Training parameters) you can see is (AdamW8bit change to AdamW) and (Gradient checkpointing, Memory efficient attention) is two also tick it
> 
> Hope it helps you

Thanks a lot, that helped ... I though it was the CUDA version or something like this ...
And changing it to AdamW still benefit from the acceleration of having RTX 3000 or 4000 series ?
